alpine-consul/root/usr/bin/consul-join

```diff
@@ -1,16 +1,13 @@
 #!/usr/bin/with-contenv sh
 
-# Get the IP of the current container. We don't want this container to match itself.
-HOST_IP=`container-ip`
+# Get the consul IPs
+JOIN=`container-ip consul | awk '{ printf " -retry-join " $0}'`
```

Comment:
I think it would be better if this script returned only the IP addresses. That will make it more reusable in other scenarios (can't think of one yet). We can keep the `-retry-join` argument to the `run` script where it is required.

What do you think?